Lib-DND5e/dnd5e_Property_getConditionEffects.command

```diff
@@ -0,0 +1,54 @@
+[h: effectsMap = json.set ("", "_blind", "Can't see; fails checks requiring sight",
```

Comment:
Need to remove the _ keys. This effectively commented them out, so they should just be removed.